src/actions.js

```diff
@@ -26,31 +26,39 @@ module.exports = {
   "batch": function(config) {
     var json = getFile(config.options.batch);
     var batch = getBatch(json);
+    const example = batch[0];
```

Comment:
Can a mumuki exercise have no examples at all? Those are mapped directly as batch entries so this will fail in that case. If it doesn't make sense, I'd handle empty batches and throw and an error anyway, just in case

Reply:
Nice suggestion! Done!